source/ExperimentConfig.h

```diff
@@ -15,7 +15,7 @@ class ExperimentConfig : public FpsConfig {
 	String serverAddress = "";							///< Address for server
 	int serverPort = 12345;								///< Port for server to listen to
 	int clientPort = 12350;								///< Port for the client to listen to
-	int numPlayers = 2;							///< Number of connections to wait for before starting the game
+	int numPlayers = 2;									///< Number of connections to wait for before starting the game
```

Comment:
This appears to have been added but not used anywhere. I think this config is likely more useful than hard coded "per client" configs. More on this below

Reply:
It is used to check how many clients have connected before starting a session. All the(2 here) clients need to be connected and ready to start a session.

Reply:
Ah okay I see this wasn't added here, looks like another whitespace-only change that could be reverted. My apologies.